### DISCUSSION TITLE
- why my html tags inside codeblock not being rendered properly, I'm using lib from npm

### DISCUSSION THREAD

**Discussion by ArnavK-09** [nuejs/nue#228] [Q&A] [answered]
![image0.jpg](https://github.com/nuejs/nue/assets/69188140/31d4a26a-bd25-4151-a437-9c1bdc84063d)

**Comment by nobkd** [comment, marked as answer]
It looks like you are using `nuxt-glow` on its own. Can you provide a small code example/reproduction?

**Maybe you already pass an escaped version of HTML code to the highlighter?**
(as the `&` and `;` are highlighted differently than `lt` and `gt`)

`&lt;` == lower than == `<` and `&gt;` == greater than == `>`.
These characters are typically encoded for code blocks, to enforce, that the `<` or `>` is text and not part of HTML markup. ⇾ Prevent accidental rendering of code as HTML

**Reply by ArnavK-09** [reply]
ohh you are right , my makrdown is encoding it, thanks
